extensions/microsoft-authentication/src/node/authProvider.ts

```diff
@@ -233,7 +233,6 @@ export class MsalAuthProvider implements AuthenticationProvider {
 				const session = this.sessionFromAuthenticationResult(result, scopeData.originalScopes);
 				this._telemetryReporter.sendLoginEvent(session.scopes);
 				this._logger.info('[createSession]', `[${scopeData.scopeStr}]`, 'returned session');
-				this._onDidChangeSessionsEmitter.fire({ added: [session], changed: [], removed: [] });
```

Comment:
This is redundant since the `update()` in the other file change will trigger this event.